cache/edge_lambdas/src/redirector.test.ts

```diff
@@ -98,6 +98,12 @@ describe('Query string redirects', () => {
       from: '/works?search=images',
       to: '/search/images',
     },
+    {
+      description:
+        'Image search within works should redirect to Image search hub page',
+      from: '/works?search=images&images.color=red', // Test legacy color param
+      to: '/search/images?images.color=red',
```

Comment:
hmmm this should actually be `color=red` or it won't work. Looking into it.

Reply:
Fixed here https://github.com/wellcomecollection/wellcomecollection.org/pull/9300/commits/8c8810fb8046b77e2261326f56398150d9074522